sharktank/sharktank/layers/paged_attention.py

```diff
@@ -131,9 +132,20 @@ def paged_attention_kv_cache_gather(
 kv_cache_gather = KVCacheGatherKernel()
 
 
-def unpack_raw_tensor(tensor):
+def unpack_to_raw_tensor(tensor: AnyTensor) -> AnyTensor:
```

Comment:
I think the initial commit also includes qualms about the name. I think raw might just be the wrong word and the whole name could just be "maybe_unpack_tensor"

Reply:
I'd assume that a maybe_unpack_tensor has a chance of failing and returning None, or something similar. I think the unconditional implication is fine. It's true of "unbox_tensor" already that we use everywhere.